src/index.tsx

```diff
@@ -80,6 +80,12 @@ export type NextTopLoaderProps = {
    *
    */
   showAtBottom?: boolean;
+  /**
+   * To show the TopLoader for hash anchors.
+   * @default true
+   *
+   */
+  showForHashAnchor?: boolean;
```

Comment:
Decided to call it `hashAnchor` to stay in line with other terminology, although `fragment` is the proper term.

Reply:
I agree with you, fragment is more proper term as it is easier to recall it and there can be different types for fragments. But right now, we can leave it as it is.